src/mango/src/mango_idx_view.erl

```diff
@@ -298,6 +304,8 @@ indexable({[{<<"$gt">>, _}]}) ->
     true;
 indexable({[{<<"$gte">>, _}]}) ->
     true;
+indexable({[{<<"$exists">>, _}]}) ->
```

Comment:
What's the intuition about making `$exists` indexable as well?

Reply:
I did not describe this intuition in a comment (perhaps I should have) but in the corresponding commit message: _"This is required to make index selection work better with covering indexes.  The `$exists` operator prescribes the presence of the given field so that if an index has the field, it shall be considered because it implies true.  Without this change, it will not happen, but covering indexes can work if the index is manually picked."_

The typical example of that is a query like `{"company": {"$exists": true}, "manager": true}`.  Without this change, only an index on `manager` could fit, not the combined index of `company_and_manager`, where both the `company` and `manager` fields are there and could have been used to serve the query.

Reply:
That makes sense. Thanks for explaining.